docs/user/grading.rst

```diff
@@ -0,0 +1,36 @@
+Grading
+=======
+
+Artemis allows instructors to define grading keys for courses and exams.
+When the grading key is defined, the total points obtained by a student from all exercises are calculated and mapped to a grade.
+Grading Keys can be either:
+
+- ``Grade`` type which maps the points to a letter or numeric grade with a ``First Passing Grade``. This can be used e.g., for final exams or courses without exams that are graded solely by exercises.
+- ``Bonus`` type which maps the points to a numeric value. This can be used e.g., when the grade obtained in the given course or exam is not an end result but complements a final exam.
+
+Instructors
+-----------
+Instructors can export and import grading keys in CSV format to reuse them inside or outside of Artemis.
+If a grading key is defined, exporting student results includes the obtained grade information as well.
+It is also possible to create or modify grading keys after an exam or course is over.
+In that case student grades will be adjusted automatically.
+
+    .. note::
+        You can check `how to create grading keys for courses and exams</user/exams/instructors_guide/#grading-key>`__ for more detailed information.
```

Comment:
The backticks are also visible in the render, don't know if this is intended or not.
<img width="51" alt="image" src="https://user-images.githubusercontent.com/25747019/179737800-44ec8da0-a957-4c78-83a5-e0cfa2ef4f01.png">

Reply:
Thanks for the catch, I fixed them now and also resolved the merge conflicts.